pkg/infrastructure/gcp/clusterapi/iam.go

```diff
@@ -0,0 +1,149 @@
+package clusterapi
+
+import (
+	"context"
+	"fmt"
+	"time"
+
+	"github.com/sirupsen/logrus"
+	resourcemanager "google.golang.org/api/cloudresourcemanager/v1"
+	iam "google.golang.org/api/iam/v1"
+	"google.golang.org/api/option"
+
+	gcp "github.com/openshift/installer/pkg/asset/installconfig/gcp"
+)
+
+// ServiceAccount predefined roles. Roles are described here https://cloud.google.com/iam/docs/understanding-roles#predefined_roles
+var masterRoles = []string{
```

Comment:
```suggestion
var masterRoles = [...]string{
```

Reply:
Move this to the function to make more readable and prevent mutation.